catroid/src/main/res/values-sw/strings.xml

```diff
@@ -1753,7 +1753,7 @@ needs read and write access to it. You can always change permissions through you
     <string name="formula_editor_logic_comparison">vikokotoa mfanano</string>
     <string name="formula_editor_object_general">Sifa za jumla</string>
     <string name="formula_editor_object_movement">mwendo mali</string>
-    <string name="formula_editor_device_visual_detection">Visual sensors</string>
+    <string name="formula_editor_device_face_detection">Visual sensors</string>
```

Comment:
undo this changes, nothing to do here, it will eventually be overwritten by the translations